### DISCUSSION TITLE
c64

### DISCUSSION THREAD

**Discussion by datamaskinen** [curl/curl#12536] [Q&A] [answered]
### I did this

I am trying to find cc65 in documentation but can not find any documentation... is it even possible to compile it for c64?

I have aswell contacted the developer of c64os (Gregorio Naçu) to get this bug fixed ... he might have some input to give on the issue...

If not ... how can we fix this bug?

### I expected the following

Have a meeting and discuss this issue... to see if this even is possible to fix ... 

### curl/libcurl version

All versions

### operating system

C64 and c64os

**Comment by dfandrich** [comment, marked as answer]
An absolute minimal curl binary is about 133 KB in length on i386 (at least,
that's what it was a few years ago) and it takes much more RAM than that when
running.  That's many times more the entire address space of a 6502. It
also assumes that it can access contiguous buffers more than 64 KiB in length,
which the 6502 can't handle.  Theoretically, you could make it fit with some
kind of an overlay system and a REU, or by running a VM on the 6502, but that
wouldn't be easy or fast. cc65 also has severe limits on the size of variables
on the stack that means it would throw errors when trying to compile curl. So,
in short, no it won't work and realistically, it never will.

**Comment by datamaskinen** [comment]
Ok... then I'll give up this... thanks for good aswer.
